config/cesm/machines/config_machines.xml

```diff
@@ -2338,6 +2338,89 @@ This allows using a different mpirun command to launch unit tests
     </module_system>
   </machine>
 
+  <machine MACH="zeus">
+    <DESC> CMCC Lenovo ThinkSystem SD530, os is Linux, 36 pes/node, batch system is LSF</DESC>
+    <NODENAME_REGEX>.*-ib</NODENAME_REGEX>
```

Comment:
Is it possible to be more specific in this regex?  This may be too general.

Reply:
I tried to introduce a more specific REGEX:
<NODENAME_REGEX>(login[1,2]-ib|n[0-9][0-9][0-9]-ib)</NODENAME_REGEX>
is it fine?